cmd/nvidia-dra-controller/imex.go

```diff
@@ -155,8 +186,35 @@ func (m *ImexManager) Stop() error {
 	return nil
 }
 
+// addImexDomain adds an IMEX domain to be managed by the ImexManager.
+func (m *ImexManager) addImexDomain(imexDomain string) error {
+	imexDomainID, cliqueID, err := splitImexDomain(imexDomain)
+	if err != nil {
+		return fmt.Errorf("error splitting IMEX domain '%s': %v", imexDomain, err)
+	}
```

Comment:
```suggestion
	if err != nil {
		err = fmt.Errorf("error splitting IMEX domain '%s': %v", imexDomain, err)
		return & transientError{false,err}
	}
```

Reply:
This is the main point of my comment; we also need to wrap this func error in case the label has a malformed value

Reply:
But this isn't a transient error, so I just don't wrap it.